posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,452 @@
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.12-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.12-beta,)'
+}
+----
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.0</feature>
+  </featureManager>
+----
+
+[#java15]
+== Support for Java SE 15
+
+Any official Java SE 15 release from link:https://adoptopenjdk.net?variant=openjdk15&jvmVariant=openj9[AdoptOpenJDK], link:https://jdk.java.net/15/[Oracle], or other OpenJDK vendor will work with Open Liberty. Java SE 15 is not a long-term supported release, with standard support scheduled to end in March 2021.
+
+Keep in mind, Eclipse OpenJ9 link:{url-prefix}/blog/2019/10/30/faster-startup-open-liberty.html[typically offers faster startup times] than Hotspot.
+
+The primary features added in this release include:
+
+* link:https://openjdk.java.net/jeps/379[JEP 379] Shenandoah: A Low-Pause-Time Garbage Collector
+* link:https://openjdk.java.net/jeps/377[JEP 377] ZGC: A Scalable Low-Latency Garbage Collector
+* link:https://openjdk.java.net/jeps/378[JEP 378] Text Blocks
+* link:https://openjdk.java.net/jeps/384[JEP 384] Records (Second Preview)
+* link:https://openjdk.java.net/jeps/360[JEP 360] Sealed Classes (Preview)
+
+For more information on downloading a version of Java 15, please visit link:https://adoptopenjdk.net/index.html?variant=openjdk15&jvmVariant=openj9[AdoptOpenJDK.net], link:https://www.eclipse.org/openj9/[Eclipse.org] or link:https://openjdk.java.net/groups/hotspot[OpenJDK.java.net].
+
+For working with the `server.env` file in Open Liberty, please visit the `Configuration Files` section of the Open Liberty link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html[Server Configuration Overview documentation].
```

Comment:
"please visit the" to "see the" (we don't need to say please in this kind of info)